nebula-components/base/_base.scss

```diff
@@ -0,0 +1,4 @@
+hr {
+  margin-top: ($nb-spacing-unit * 2);
+  border-color: $nb-color-grey-light;
+}
```

Comment:
We probably want to take this away so it doesn't accidentally ship.  We will look at tidying up the demo pages once the navigation is done and we can do some routing.